crates/kornia-apriltag/src/family/mod.rs

```diff
@@ -0,0 +1,22 @@
+/// Represents the AprilTag Family
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct TagFamily {
+    /// The width of the tag including the border, in units.
+    pub width_at_border: usize,
+    /// Whether the border is reversed.
+    pub reversed_border: bool,
+    /// The total width of the tag, including border, in units.
+    pub total_width: usize,
+    /// The number of bits in the tag code.
+    pub nbits: usize,
+    /// The x-coordinates of each bit in the tag.
+    pub bit_x: &'static [u8],
+    /// The y-coordinates of each bit in the tag.
+    pub bit_y: &'static [u8],
+    /// The code data for the tag family.
+    pub code_data: &'static [usize],
```

Comment:
probably is worth to have owned data here as result and we avoid lifetime parameters